automatic/opera/tools/helpers.ps1

```diff
@@ -1,13 +1,3 @@
-function Is32BitInstalled {
```

Comment:
Is there no longer any way to detect if the 32bit version is installed?

Reply:
There is no need to do so. `force86` is remembered now so it will be reused in subsequent install.  Also, this is not specific to this package.

Reply:
I was under the impression that it only remembered the previous install/update.
Maybe that was fixed though without me noticing.

Reply:
:point_up: [September 21, 2017 8:53 PM](https://gitter.im/chocolatey/chocolatey-coreteampackages?at=59c40a8ebac826f054e5902f)

Reply:
Oh right, completely forgot about that one.
Ignore me on this one then...